Corona-Warn-App/src/test/java/de/rki/coronawarnapp/eventregistration/checkins/split/CheckInSplitterTest.kt

```diff
@@ -150,9 +136,9 @@ class CheckInSplitterTest : BaseTest() {
     }
 
     @Test
-    fun `Scenario 6`() {
+    fun `Scenario 5`() {
         /*
-        // Example 6 - 2-days-different-months check-in
+        // Example 5 - 2-days-different-months check-in
```

Comment:
very minor: How about naming the test accordingly instead and dropping the comment?

Reply:
good idea!

Reply:
renamed it!